File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-api/src/main/java/com/cbi/ccr/csw/dashboard/mqprimitive/dto/MQPrimitiveDTO.java
package com.cbi.ccr.csw.dashboard.mqprimitive.dto;

import com.cbi.frw.api.dto.GenericDTO;

import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class MQPrimitiveDTO extends GenericDTO{
	
	private String id;
	private String mqChannel;
	private String primitive;
	private String queueName;
	private Boolean toLoad;
	

}

File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-ui/src/app/modules/configuration/mq-primitive/dto/mq-primitive-dto.ts
import { CommonConfigDTO, CommonConfigFilterDTO } from "../../common-config-dto";

export class MQPrimitiveFilterDTO extends CommonConfigFilterDTO{

    mqChannel!: string;
    primitive!: string;
    queueName!: string;
   
}

export class MQPrimitiveDTO extends CommonConfigDTO{

    mqChannel!: string;
    primitive!: string;
    queueName!: string;
    toLoad!: boolean;
 
}